numpy/random/_generator.pyx

```diff
@@ -665,6 +665,13 @@ cdef class Generator:
         array([3,1,0]) # random
         >>> #This is equivalent to rng.permutation(np.arange(5))[:3]
 
+        Generate a uniform random sample from a 2-D array, without
+        replacement:
```

Comment:
I would prefer a wording that emphasizes that it chooses along the first axis. This wording can be read to mean that it selects values from all of the elements of the 2D array (i.e. equivalent to `choice(x.ravel())`).